slack.go

```diff
@@ -57,11 +57,12 @@ type authTestResponseFull struct {
 type ParamOption func(*url.Values)
 
 type Client struct {
-	token      string
-	endpoint   string
-	debug      bool
-	log        ilogger
-	httpclient httpClient
+	token         string
+	appLevelToken string
```

Comment:
I'm sorry. I have one more question.
Is it better to set the app level token and normal token separately?Currently there is no distinction between bot and user tokens, but I don't think this is a problem because they have a lot in common.

Reply:
I'm assuming that there will be a fair number of cases in which someone would want to reuse a single client with both an OAuth token and an app-level token. App-level tokens are completely different from standard bot/user tokens, and I would lean towards keeping it as a separate parameter.

It also helps clear up confusion where a user might try to call the `ListEventAuthorizations` without being aware that they're using the wrong token.

If you think it should be one parameter, though, I'll definitely make that change!

Reply:
Thanks! Looks good to me!